### ISSUE TITLE
Looking to update Proxmark3 client version from v4.9237 to v4.13441

### ISSUE THREAD

**Issue by ASchneider-GitHub** [RfidResearchGroup/RFIDtools#73]
Hey there. I'm currently looking to update RFIDtools and have it run the newest version of the PM3 client, which appears to be v4.13441 (released 3 days ago). I'm not seeing documentation for switching client versions before compiling the application. Is this something that can be done on my end, or is it going to require a lift from you guys to make sure everything is compatible?

**Comment by F9Alejandro** [comment]
The app would have to undergo a massive update to have its client interpreter up to the same version as the fw on the proxmark3 since it doesn't handle mismatch very well.

**Comment by ASchneider-GitHub** [comment]
@F9Alejandro So in that case is it better to downgrade the bootrom and fullimage files to match the client version running on the RFIDtools application? Or is it okay to have the client at v4.9237, and the PM3 device at v4.13441?

**Comment by F9Alejandro** [comment]
You wouldn't want to downgrade the firmware or the image due to the massive changes between versions. It isn't smart to go back as you loose the ability and functions that came with the release. Best bet is to wait for someone to come around and update the app or to try using the termux version of the client once it is up to date.

Patients is a virtue in this case as you don't want to rush something and brick/break something.

**Comment by ASchneider-GitHub** [comment]
Makes sense. I just wanted to make sure it was okay to keep the PM3 device & my Proxspace installation on v4.13441, even though RFIDtools is running on v4.9237. I do notice some pretty big functionality differences between the most recent version of PM3 and the one that RFIDtools is running which is unfortunate since I was hoping to use the PM3 device on the fly, but since there are over 4000 commits between the majour version releases, I'd imagine the RFIDtools' client version is pretty outdated as far as features go.

**Comment by F9Alejandro** [comment]
Last update for its client/fw bundle was last year around June. So def outdated since there are almost daily commits going on.

**Comment by UltrashRicco** [comment]
In my case, FW v4.13441 (https://github.com/RfidResearchGroup/proxmark3/releases/tag/v4.13441) is the only one that works with my Proxmark Easy (256kB) with RFIDtools.

v4.9237 does not work. Firmwares more recent than v4.13441 do not work either. All give the following error:
Capabilities structure version sent by Proxmark3 is not the same as the one used by the client!
Please flash the Proxmark with the same version as the client.
ERROR: cannot communicate with the Proxmark

Is there any active development on the RFIDtools APP? I would love to help, but I know very little about programming.
